pcs/cli/resource/parse_args.py

```diff
@@ -50,16 +50,15 @@ def parse_clone(arg_list, promotable=False):
         raise CmdLineInputError(
             "op settings must be changed on base resource, not the clone",
         )
-    parts["meta"] = prepare_options(
-        groups.get("options", []) + groups.get("meta", []),
-    )
+
+    if "meta" in groups:
+        raise CmdLineInputError(
+            "meta options is not supported in 'pcs resource clone' command"
+        )
+
+    parts["clone_option"] = prepare_options(groups.get("options", []))
     if promotable:
-        if "promotable" in parts["meta"]:
-            raise CmdLineInputError(
-                "you cannot specify both promotable option and promotable "
-                "keyword"
-            )
```

Comment:
This check and the related exception is needed. It prevents command `pcs resource promotable resource-name promotable=false` to pass, as it is a mistake to create a promotable clone and set its promotable flag to false in one step.